conda-env/dev.yml

```diff
@@ -0,0 +1,31 @@
+# Conda pcmdi_metrics development environment
+name: pcmdi_metrics_dev
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  # Base
+  # ==================
+  - python=3.8.10
```

Comment:
Note, `python=3.9.7` causes package conflicts with another package that doesn't support Python>=3.9. It is most likely one of the CDAT packages.

Reply:
Package conflicts were caused by one of these packages: `cdat::vcs`, `cdat::vcsaddons`, and `PCMDI::cia`.
Removing them has solved this issue.